nrrd/reader.py

```diff
@@ -322,8 +321,10 @@ def read_data(header, fh=None, filename=None, index_order='F'):
         Filename of the header file. Only necessary if data is detached from the header. This is used to get the
         absolute data path.
     index_order : {'C', 'F'}, optional
-        Specifies the index order of the resulting data array. Either 'C' (C-order) where the dimensions are ordered from
-        slowest-varying to fastest-varying (e.g. (z, y, x)), or 'F' (Fortran-order) where the dimensions are ordered
+        Specifies the index order of the resulting data array.
```

Comment:
This seems to have broken at weird spots. The line length is way under 120. Any idea what happened there?

Reply:
I think I broke it manually, let me fix it